packages/datagateway-dataview/src/state/app.types.tsx

```diff
@@ -10,6 +14,7 @@ export interface DGDataViewState {
   settingsLoaded: boolean;
   selectAllSetting: boolean;
   pluginHost: string;
+  isisDatafilePreviewer: DatafilePreviewerState;
```

Comment:
small point, but we could probably rename this to be more generically `datafilePreviewer` as it's not technically ISIS specific - it's just singe level IDS specific